environment.yml

```diff
@@ -15,7 +15,7 @@ dependencies:
     - litex-hub::prjxray-tools=0.1_2842_g6867429c=20210301_104249
     - litex-hub::prjxray-db=0.0_252_g8372b58=20210622_220029
     - litex-hub::yosys=0.9_5457_gc6681508=20210615_141355_py38
-    - litex-hub::nextpnr-fpga_interchange=v0.0_3610_gc73d4cf6=20210615_141355
+    - litex-hub::nextpnr-fpga_interchange=v0.0_3644_g8a9fb810=20210625_074838
```

Comment:
Is this the version with all the required features/fixes which is required?

Reply:
Yes, this one fixes the physical netlist generation, and has the two fixes mentioned above, plus https://github.com/YosysHQ/nextpnr/pull/748